app/src/main/java/pl/ludek/smat/home/pool_office_client/domain/model/PoolInfoData.kt

```diff
@@ -0,0 +1,3 @@
+package pl.ludek.smat.home.pool_office_client.domain.model
+
+data class PoolInfoData(var t1: Float? =null, var t2: Float? =null, var t3: Float? =null, var p1: Float? =null)
```

Comment:
Avoid using nullable values in data classes. Is it possible that any of those values is null?

Reply:
Corrected. I just thought of using this view because I wanted to first use some kind of content provider, something like dependency injection in spring or a content provider in react. But then I decided that these are superfluous, here the program is not so complicated.